app/styles/site-layouts.scss

```diff
@@ -1,3 +1,22 @@
-body {
- padding: 10px;
+$app-qgreen: rgba(98,144,128,0.25);
+$app-halfgreen: rgba(98,144,128,0.5);
+
+.search-button {
+  color: $app-halfgreen;
 }
+
+.text-icon {
+  text-decoration-color: $app-qgreen;
+  font-size: 31.5px;
```

Comment:
try to save font-sizes as constants too.

Reply:
done